### PR TITLE
Update Catalan locales

### PR DESCRIPTION
**Pull request by avizcaino**
I've updated the catalan locales and uncommented the corresponding imports. Also fixes the tests.

### CODE REVIEW

File: src/locale/ca/_lib/match/index.js [hunk 1]
```diff
@@ -1,47 +1,151 @@
...
+    /^des/i
+  ]
 }
 
-var matchTimesOfDayPatterns = {
-  short: /^(am|pm)/i,
-  long: /^([ap]\.?\s?m\.?)/i
+var matchDayPatterns = {
+  narrow: /^(dg|dl|dt|dm|dj|dv|ds)/i,
+  short: /^(dg.|dl.|dt.|dm.|dj.|dv.|ds.)/i,
```
Comment (leshakoss) [inline src/locale/ca/_lib/match/index.js]: Please replace dots with `\.`

File: src/locale/ca/_lib/localize/index.js [hunk 2]
```diff
@@ -1,50 +1,194 @@
...
+  wide: ['Abans de Crist', 'Després de Crist']
 }
 
+var quarterValues = {
+  narrow: ['1', '2', '3', '4'],
+  abbreviated: ['T1', 'T2', 'T3', 'T4'],
+  wide: ['1er trimestre', '2on trimestre', '3er trimestre', '4rt trimestre']
+}
+
+// Note: in English, the names of days of the week and months are capitalized.
```
Comment (leshakoss) [inline src/locale/ca/_lib/localize/index.js]: This comment is meaningless in this locale, please remove it

File: src/locale/ca/index.js [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
+import localize from './_lib/localize/index.js'
+import match from './_lib/match/index.js'
 
 /**
  * @type {Locale}
  * @category Locales
  * @summary Catalan locale.
  * @language Catalan
  * @iso-639-2 cat
- * @author Guillermo Grau [@guigrpa]{@link https://github.com/guigrpa}
```
Comment (leshakoss) [inline src/locale/ca/index.js]: Please don’t remove the previous contributor

File: src/locale/ca/_lib/match/index.js (hunk 1 above)
Comment (avizcaino) [reply src/locale/ca/_lib/match/index.js]: Done!

File: src/locale/ca/_lib/localize/index.js (hunk 2 above)
Comment (avizcaino) [reply src/locale/ca/_lib/localize/index.js]: Done!

File: src/locale/ca/index.js (hunk 3 above)
Comment (avizcaino) [reply src/locale/ca/index.js]: Sorry, my bad. Fixed!

File: src/locale/ca/index.js [hunk 4]
```diff
@@ -11,19 +11,20 @@
...
+    weekStartsOn: 1 /* Monday */,
+    firstWeekContainsDate: 4
+  }
+}
 
-// export default locale
+export default locale
 
-throw new Error('ca locale is currently unavailable. Please check the progress of converting this locale to v2.0.0 in this issue on Github: TBA')
+// throw new Error('ca locale is currently unavailable. Please check the progress of converting this locale to v2.0.0 in this issue on Github: TBA')
```
Comment (kossnocorp) [inline src/locale/ca/index.js]: Please remove this line

File: src/locale/ca/_lib/localize/index.js [hunk 5]
```diff
@@ -1,50 +1,193 @@
...
+  wide: ['Abans de Crist', 'Després de Crist']
 }
 
+var quarterValues = {
+  narrow: ['1', '2', '3', '4'],
+  abbreviated: ['T1', 'T2', 'T3', 'T4'],
+  wide: ['1er trimestre', '2on trimestre', '3er trimestre', '4rt trimestre']
+}
+
+// If you are making a new locale based on this one, check if the same is true for the language you're working on.
```
Comment (leshakoss) [inline src/locale/ca/_lib/localize/index.js]: You've deleted just the first line of the comment. Could you please delete the rest? 😄

Comment (avizcaino) [reply src/locale/ca/_lib/localize/index.js]: done!

**Comment by rubenpro** [comment]
Hi there, why is this locale not merged yet? We need it!

**Comment by kossnocorp** [comment]
I've released the change as `v2.0.0-alpha.29`: https://gist.github.com/kossnocorp/a307a464760b405bb78ef5020a4ab136#v200-alpha29

**Pull request merged**